test/core/tests/tag_catch.py

```diff
@@ -121,7 +121,9 @@ def check_results(self, flow, checker):
                     data = task.data
                     got = sorted(m.value for m in task.metadata if m.type == "attempt")
                     if flow._graph[step.id].parallel_step:
-                        if "control" in task.id:
+                        if task.metadata_dict.get(
```

Comment:
can you pull this out as a separate PR - this change is likely non-contentious.

Reply:
Done https://github.com/Netflix/metaflow/pull/1917